llpc/context/llpcPipelineContext.cpp

```diff
@@ -224,6 +224,7 @@ void PipelineContext::setPipelineState(Pipeline *pipeline, bool unlinked) const
       lgcStageMask |= 1 << getLgcShaderStage(static_cast<ShaderStage>(stage));
   }
   pipeline->setShaderStageMask(lgcStageMask);
+  pipeline->set128BitCacheHash(get128BitCacheHashCode(), {LLPC_INTERFACE_MAJOR_VERSION, LLPC_INTERFACE_MINOR_VERSION});
```

Comment:
Why are you including the LLPC interface version in the metadata? If the idea is to ensure that two different driver builds have different numbers, I don't think it's enough for that.

Reply:
The elf is generated with a particular version of LLPC.  The cache creator tools build a cache for a particular driver that uses a particular version of LLPC.  If those two versions of LLPC do not match, then the cache could potentially be useless.  If the hashing algorithm changed, then the cache will miss all of the time.  This gives us an early warning that the cache will be useless.  I understand that to be an interface change.  Correct me if I am wrong.

Reply:
You could have two different driver builds with different LLPC code generating different elves, but they have the same LLPC interface major version. Is that a problem for you? Do you have protection for that somewhere else?

Reply:
I recognized that the different ELF could be a problem.  Assuming the ELF files are correct, you will still get correct behaviour.  We are more interested in avoiding loading ELF for a different shader or pipeline.  We can talk about this more in another setting if you would like.